File: 11508_2+1세일/첫번째 시도_성공.cpp
#include <iostream>
#include<vector>
#include<algorithm>
using namespace std;

int main() {
	vector<int> input;
	int testcase;
	int sum_output = 0;
	cin >> testcase;
	for (int i = 0, num; i < testcase; i++) {
		cin >> num;
		input.push_back(num);
	}

	sort(input.begin(), input.end());
	for (int i = 1; i <= testcase; i++) {
		if (i % 3 != 0) {
			sum_output += input[testcase - i];
		}
	}

	cout << sum_output;
	return 0;
}

File: Backjoon/1758_알바생 강호/첫번째 시도_성공.py
testcase = int(input())
num = []
for i in range(testcase):
    intput = int(input())
    num.append(intput)

num.sort(reverse = True)
sum_output = 0
for i in range(0,len(num)):
    if num[i] - i > 0:
        sum_output+= num[i]-i
        
sum_output
